### PR TITLE
Update readme for null safety

### PR DESCRIPTION
**Pull request by incendial**
### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix
[ ] New rule
[ ] Changes an existing rule
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,10 @@
 * Changed the support version range of the `analyzer_plugin` to `>=0.3.0 <0.5.0`
```
Comment (dkrutskikh) [inline CHANGELOG.md:8]: @incendial please remove or update to ^0.5.0

File: README.md [hunk 2]
```diff
@@ -1,48 +1,72 @@
...
+
+    ```
+
+    **OR**
+
+    add it manually to `pubspec.yaml`
 
     ```yaml
     dev_dependencies:
       dart_code_metrics: ^2.4.1
```
Comment (dkrutskikh) [inline README.md:58]: @incendial please up to 3.0.0

File: README.md [hunk 3]
```diff
@@ -118,23 +164,141 @@ Usage: metrics [arguments...] <directories>
...
+
+dart_code_metrics:
+  anti-patterns:
+    - long-method
+    - long-parameter-list
+  metrics:
+    cyclomatic-complexity: 20
+    lines-of-executable-code: 50
+    number-of-arguments: 4
+    maximum-nesting: 5
```
Comment (dkrutskikh) [inline README.md:206]: @incendial `maximum-nesting-level`

**Review by dkrutskikh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (incendial) [reply CHANGELOG.md:8]: Let's do that in the null safety migration request

File: README.md (hunk 3 above)
Comment (incendial) [reply README.md:206]: done

**Review by dkrutskikh — APPROVED** [review]

**Pull request merged**